api/client/webclient/webconfig.go

```diff
@@ -72,10 +72,14 @@ type WebConfigAuthSettings struct {
 	Providers []WebConfigAuthProvider `json:"providers,omitempty"`
 	// LocalAuthEnabled is a flag that enables local authentication
 	LocalAuthEnabled bool `json:"localAuthEnabled"`
+	// PwdlessEnabled is true if passwordless logins are allowed.
+	PwdlessEnabled bool `json:"pwdlessEnabled,omitempty"`
```

Comment:
Maybe:

```suggestion
	// AllowPasswordless is true if passwordless logins are allowed.
	AllowPasswordless bool `json:"allowPasswordless,omitempty"`
```

so it matches the lingo of the ping messages? I would usually avoid abbreviations in public APIs as well.